### ISSUE TITLE
Add verbose flag

### ISSUE THREAD

**Issue by alimgiray** [codersrank-org/multi_repo_extractor#1]
It can be useful to add a "--verbose" flag to show more data about the processing status. 

Besides adding more logs, we can also redirect git clone/pull output and repo_info_extractor output to the stdout.

**Comment by SolKuczala** [comment]
Hi, if you don't mind I'll take a look to see if I can do something :) Got a question: from where you create your token to provide it to the program?

**Comment by alimgiray** [comment]
> Hi, if you don't mind I'll take a look to see if I can do something :) Got a question: from where you create your token to provide it to the program?

Hi SolKuczala, contributions are welcome, thanks :). You can create your token from this url: https://github.com/settings/tokens

**Comment by SolKuczala** [comment]
I found a couple of issues in the middle that I don't know if you want to check them or want me to write them directly in issues. I'm using it in a fish terminal in Ubuntu 18. 
- If I'm not signed up in CodersRank will do the process to end up opening the browser to the expected profile that it doesn't exist without any fail >> https://profile.codersrank.io/repo?multiToken={token}
- In a second try to change the token to something else like many spaces, will ask you to provide the email when I kept doing this:
./multi_repo_repo_extractor -token $TOKEN -emails my.email@xmail.com -repo_visibility "public"

**Comment by SolKuczala** [comment]
I have a resolution for the opened issue #3, if you can label it for hacktoberfest, I can have it for my Pr's too and I'll appreciate :)

**Comment by alimgiray** [comment]
Thank you for your feedback!

For the 1st point we already noticed that, planning to make some changes to it, but it's not really finalized. I'll open issue when it is done.

About the second one I didn't know it so thanks for pointing it out. I'll check it.

Also added label, but I checked it now, rules are changed a little bit: 
https://hacktoberfest.digitalocean.com/hacktoberfest-update 

So when you submit your PR, I'll add `hacktoberfest-accepted` label to it.
